4.4/contrib/etc/scl_enable

```diff
@@ -0,0 +1,23 @@
+# This will make scl collection binaries work out of box.
+unset BASH_ENV PROMPT_COMMAND ENV
+source scl_source enable rh-nodejs4
+
+#Set the debug port to 5858 by default.
+if [ -z "$DEBUG_PORT" ]; then
+  export DEBUG_PORT=5858
+fi
+
+# Set the environment to development by default.
+if [ -z "$DEV_MODE" ]; then
+  export DEV_MODE=false
+fi
```

Comment:
I think this code was relocated in https://github.com/openshift/s2i-nodejs/pull/89